examples/website/i3s/app-debug.js

```diff
@@ -274,7 +275,17 @@ export default class App extends PureComponent {
     this.setState({tilesetUrl, name, token});
     this.handleClearWarnings();
     const metadata = await fetch(metadataUrl).then((resp) => resp.json());
-    this.setState({metadata, tileInfo: null, normalsDebugData: []});
+    const layerUrls = await this.getLayerUrls(tilesetUrl);
+    this.setState({metadata, tileInfo: null, normalsDebugData: [], layerUrls});
+  }
+
+  async getLayerUrls(tilesetUrl) {
+    try {
+      const tileset = await load(tilesetUrl, I3SBuildingSceneLayerLoader);
+      return tileset?.sublayers.map(sublayer => sublayer.url);
+    } catch (e) {
+      return [tilesetUrl];
```

Comment:
Explain the two return value types

Reply:
Explained by JSDoc here https://github.com/visgl/loaders.gl/pull/1839